### DISCUSSION TITLE
how to deploy Flink CDC pipeline using kubernetes

### DISCUSSION THREAD

**Discussion by litZz** [apache/flink-cdc#3037] [Q&A]
flink-cdc-cli CliFrontend not getting StreamExecutionEnvironment through the standard way(StreamExecutionEnvironment.getExecutionEnvironment()), so flink application enrtypoint  EmbeddedExecutorServiceLoader not effected.

**Comment by herunkang2018** [comment]
@litZz I agree, due to this reason above, CDC pipeline job cannot be submitted by flink client, but IMO this is the main way to submit job. I think we should support this way to make it convenient for users to use CDC pipeline job.

**Comment by dataengineervishal** [comment]
Any luck on this one? Still I see this is not supported.
